js/common/grids.d.ts

```diff
@@ -597,9 +636,15 @@ export type ColumnHeaderFilter = {
    * @default undefined
    */
   height?: number;
+  /**
+   * @docid GridBaseColumn.headerFilter.search
+   * @type object
```

Comment:
Why we have @type object here?

Reply:
I took as example HeaderFilter.texts option, which has @type object

Reply:
It is not necessary here, let's remove it